scripts/accumulator-server.py

```diff
@@ -36,16 +36,47 @@
 #   "Content-Type: application/x-www-form-urlencoded" has been problematic in the pass)
 
 from flask import Flask, request, Response
-from sys import argv, exit
+from getopt import getopt, GetoptError
 from datetime import datetime
 from math import trunc
 from time import sleep
+import sys
 import os
 import atexit
 import string
 import signal
 import json
 
+def usage_and_exit(msg):
+    """
+    Print usage message and exit"
+
+    :param msg: optional error message to print
+    """
+
+    if msg != '':
+        print msg
+        print
+
+    usage()
+    sys.exit(1)
+
+
+def usage():
+    """
+    Print usage message
+    """
+
+    print 'Usage: %s -h <host> -p <port> -u <server url> -p -v -h' % os.path.basename(__file__)
+    print ''
+    print 'Parameters:'
+    print "  -h <host> (or '--host <host>'): host to use database to use (default is '0.0.0.0')"
+    print "  -p <port> (or '--port <port'): port to use (default is 1028)"
+    print "  -u <server url> (or '--url <server url>'): server URL to use (default is /accumulate)"
+    print "  -r (or --pretty-print): pretty print mode"
+    print "  -v (or --verbose): verbose"
+    print "  -g (or --help): print this usage message"
```

Comment:
To start with, not sure the short names are really necessary.
However, if we want them, perhaps we could use the word *path* instead of url and use '-P' (a but confusing to have -p and -P, but I see no better solution for now)